builder/frameworks/espidf.py

```
@@ -290,75 +292,96 @@ def get_flag(line):
     return
 
+def convert_component_name_to_filesystem(component_name):
+    """Convert component name from registry format to filesystem format"""
+    # Convert "espressif/esp32-camera" to "espressif__esp32-camera"
+    # Replace "/" with "__"
+    filesystem_name = component_name.replace("/", "__")
+    return filesystem_name
+
+def backup_pioarduino_build_py():
+    """Create backup of the original pioarduino-build.py"""
+    if not "arduino" in env.subst("$PIOFRAMEWORK"):
```

Ruff (0.11.9)

Test for membership should be `not in`

Convert to `not in`

(E713)